cs/test/core/Core.csproj

```diff
@@ -81,6 +82,7 @@
     <Compile Include="$(IntermediateOutputPath)\NamespaceConflict_types.cs" Condition="False" />
     <Compile Include="$(IntermediateOutputPath)\NamespaceConflictBond_types.cs" Condition="False" />
     <Compile Include="$(IntermediateOutputPath)\Bond File With Spaces_types.cs" Condition="False" />
+    <Compile Include="XmlSerializationTests.cs" />
```

Comment:
Minor: can you put this with the other XML file and the Json file in alpha order with the rest of the Compile items?